lib/open_project/scm/manager.rb

```diff
@@ -46,6 +46,18 @@ def enabled
           registered.select { |scm| Setting.enabled_scm.include?(scm) }
         end
 
+        ##
+        # Return a hash of all managed paths for SCM vendors
+        # { Vendor: <Path> }
+        def managed_paths
+          paths = {}
+          @scms.each do |vendor, klass|
+            paths[vendor] = klass.managed_root if klass.manageable?
```

Comment:
Not only the `enabled` vendors?

I believe in another PR of yours there is even a helper for the vendors that are both enabled and managable...